fastlane/Fastfile

```diff
@@ -465,26 +466,56 @@ platform :ios do
             UI.user_error! "missing id for tests"
         end
 
-        run_tests(
-            testplan: testplan,
-            scheme: scheme,
-            skip_detect_devices: true,
-            configuration: "Debug",
-            # test_without_building: true,
-            sdk:"iphonesimulator",
-            destination: "platform=iOS Simulator,id=#{ENV['IOS_SIM_ID']},OS=#{ios_version}",
-            # code_coverage: true,
-            derived_data_path: "DerivedData",
-            buildlog_path: build.build_path,
-            result_bundle: true,
-            output_directory: "#{build.artifact_path(with_filename: false)}/tests/#{scheme}",
-            output_types: "junit",
-            cloned_source_packages_path: ENV['PACKAGES_DIR'],
-            disable_package_automatic_updates: true,
-            skip_package_dependencies_resolution: ENV["HAS_PACKAGES_CACHE_HIT"],
-            number_of_retries: 2
-        )
+        begin
```

Comment:
split in 2, so we fail on build, but continue testing next schemes on test failures